src/lib/api.test.ts

```diff
@@ -112,19 +112,16 @@ describe('toaGetJobsRequest', () => {
 
         const result = await toaGetJobsRequest()
 
-        const mockToken = Buffer.from('testusername:testpassword').toString('base64')
         expect(global.fetch).toHaveBeenCalledWith('https://toa:67890/api/jobs', {
             method: 'GET',
             headers: {
-                Authorization: `Basic ${mockToken}`,
                 'Content-Type': 'application/json',
             },
         })
         expect(result).toEqual(mockTOAData)
     })
 
     it('should error if token not found', async () => {
-        process.env.TOA_BASIC_AUTH = ''
         await expect(toaGetJobsRequest()).rejects.toThrow('TOA token failed to generate')
     })
```

Comment:
if we're getting rid of the token, probably there should be no error if token not found